include/IVideoDriver.h

```diff
@@ -121,8 +121,9 @@ namespace video
 
         virtual bool bindGraphicsPipeline(video::IGPURenderpassIndependentPipeline* _gpipeline) = 0;
 
-        virtual bool bindDescriptorSets(E_PIPELINE_BIND_POINT _pipelineType,uint32_t _first, uint32_t _count,
-            video::IGPUDescriptorSet** _descSets, uint32_t _dynOffsetCount, const uint32_t* _dynOffsets) = 0;
+        //! WARNING: Note that elements of _dynamicOffsets will be moved-assigned!
```

Comment:
yeah maybe we should make an exception (and allow for copy ctoring), otherwise this will be bugprone 101

Reply:
resolved